server/form-pages/apply/basic-information/confirmDetails.ts

```diff
@@ -1,9 +1,22 @@
-import type { TasklistPage } from 'approved-premises'
+import type { Request } from 'express'
+import type { TasklistPage, Person } from 'approved-premises'
+
+import type { DataServices } from '../../../services/applicationService'
 
 export default class ConfirmDetails implements TasklistPage {
   name = 'confirm-details'
 
-  title = `Confirm ${this.details().names}'s details`
+  title: string
+
+  details: Person
+
+  async setup(request: Request, dataServices: DataServices): Promise<void> {
```

Comment:
I think it would make sense to add an optional `setup` method to the TasklistPage interface if multiple pages have them - but that can be done in a future PR as needed

Reply:
Yeah, I think I'll do it in another PR. Might make more sense to make it an abstract class, as you suggested before